service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrder.java

```diff
@@ -33,4 +34,9 @@
     public boolean isDraft() {
         return action == null || !action.getType().equals(SEND_TO_ALL_PARTIES);
     }
+
+    @JsonIgnore
+    public boolean isInJudgeReview() {
+        return status != null && status.equals(JUDGE_REVIEW);
```

Comment:
can simplify to just `status == JUDGE_REVIEW` as we are dealing with enums

Reply:
james has resolved this one